app/models/project.rb

```diff
@@ -50,6 +51,11 @@ def budget_status
     budget - hours.sum(:value) if budget
   end
 
+  def update_user_activity
+    user_activity = DateTime.now
+    save
+  end
```

Comment:
If you use the timestamp fields to consider the user activity, you change the order with a name change, for example. This change is not a signal of user activity in this project. But a new hour entry, yes. With this change, we can decide what it's event is an user activity.